config/webpack.common.js

```diff
@@ -65,18 +70,25 @@ module.exports = {
       {
         test: /\.css$/,
         use: [
-          "style-loader",
+          MiniCssExtractPlugin.loader,
           "css-loader"
         ]
       },
       {
         test: /\.scss$/,
         use: [
-          "style-loader",
+          MiniCssExtractPlugin.loader,
           "css-loader",
           "sass-loader"
         ]
       },
+      {
+        test: /\.(woff|woff2)$/i,
+        loader: "url-loader",
+        options: {
+          limit: 8192,
```

Comment:
Could you explain this magic number in comment?

Reply:
[b5fb688](https://github.com/allegro/turnilo/pull/851/commits/b5fb688ed80bb865d54cdf27b976456155bfed2b)